uiowa/uiowa
# [SiteNow DCV] Added public directory path

# How to test
```
ddev blt ds --site=sandbox.uiowa.edu &&  ddev drush @sandbox.local uli /admin/config/sitenow/dcv
```
1. Review code changes
2. Upload a txt file
3. Run ` ddev sequelpro`
4. Look at file managed table `SELECT fid, filename, filesize, uri, filemime, status FROM file_managed;` and confirm that file was uploaded to public files directory and as `0` status
